tests/functionaltests/UXTestHelpers.h

```diff
@@ -0,0 +1,91 @@
+//******************************************************************************
+//
+// Copyright (c) Microsoft. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+
+#pragma once
+
+// UX helper methods
+#import <UIKit/UIKit.h>
+
+#include <COMIncludes.h>
+#import <WRLHelpers.h>
+#import <ErrorHandling.h>
+#import <RawBuffer.h>
+#import <wrl/client.h>
+#import <wrl/implements.h>
+#import <wrl/async.h>
+#import <wrl/wrappers/corewrappers.h>
+#import <windows.foundation.h>
+#include <COMIncludes_end.h>
+
+#import "ObjCXamlControls.h"
+#import "UWP/WindowsUIXamlControls.h"
+
+UIWindow* GetCurrentWindow();
+
+NSString* NSStringFromPropertyValue(RTObject* rtPropertyValue);
+NSString* NSStringFromPropertyValue(const Microsoft::WRL::ComPtr<IInspectable>& inspPropertyValue);
+
+class ViewControllerHelper {
+public:
+    ViewControllerHelper(UIViewController* viewController) : _externalViewController(viewController) {
```

Comment:
>ViewControllerHelper(UIViewController* viewController) [](start = 4, length = 54)

would it make more sense to take an optional second argument that is the VC on which to present?  The default value could be GetCurrentWindow().rootViewController.